src/platforms/esp32/components/avm_builtins/i2c_driver.c

```diff
@@ -99,9 +105,39 @@ Context *i2c_driver_create_port(GlobalContext *global, term opts)
     struct I2CData *i2c_data = calloc(1, sizeof(struct I2CData));
     i2c_data->transmitting_pid = term_invalid_term();
 
-    term scl_io_num_term = interop_proplist_get_value(opts, SCL_IO_NUM_ATOM);
-    term sda_io_num_term = interop_proplist_get_value(opts, SDA_IO_NUM_ATOM);
-    term clock_hz_term = interop_proplist_get_value(opts, I2C_CLOCK_HZ_ATOM);
+    term scl_io_num_term = interop_kv_get_value_default(opts, scl_io_num_atom, term_invalid_term(), global);
```

Comment:
let's rely on the default.

Reply:
What default?  Isn't that device-specific?

Reply:
There is no default for SCLK GPIO pins.